tests/thicc-hm.rn.json

```diff
@@ -0,0 +1 @@
+{"code": 0, "stdout": "[{'id', 'name', 'username', 'email', 'address', 'phone', 'website', 'company'}, {'id', 'name', 'username', 'email', 'address', 'phone', 'website', 'company'}, {'id', 'name', 'username', 'email', 'address', 'phone', 'website', 'company'}, {'id', 'name', 'username', 'email', 'address', 'phone', 'website', 'company'}, {'id', 'name', 'username', 'email', 'address', 'phone', 'website', 'company'}, {'id', 'name', 'username', 'email', 'address', 'phone', 'website', 'company'}, {'id', 'name', 'username', 'email', 'address', 'phone', 'website', 'company'}, {'id', 'name', 'username', 'email', 'address', 'phone', 'website', 'company'}, {'id', 'name', 'username', 'email', 'address', 'phone', 'website', 'company'}, {'id', 'name', 'username', 'email', 'address', 'phone', 'website', 'company'}]\n", "stderr": ""}
```

Comment:
I missed this test while reviewing. The `HashMap` object is not properly working with `Array` type. It is working normally but not working inside `Array` type.

Reply:
I think the issue is with `core/datatypes.py` line 673 to 677 at `Array` method `__str__` and `__repr__`.